<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_kta</class>
 <widget class="QDialog" name="Dialog_kta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kreistasche</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>881</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Seite 1</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="2">
       <widget class="QLineEdit" name="lineEdit_gegenlauf"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mittelpunkt in X</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Taschentiefe</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>[DM]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_dm"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>[TT]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_y"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Zustellmaß</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_x"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="lineEdit_zust"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>[WKZID]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Mittelpunkt in Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>[MX]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Durchmesser</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit_ti"/>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_34">
        <property name="text">
         <string>[ZU]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox_wkz"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Werkzeug</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>[MY]</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_39">
        <property name="text">
         <string>Gegenlauf</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>[GL]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_bild">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>461</width>
       <height>381</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Seite 2</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>9</y>
       <width>391</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="2">
       <widget class="QLineEdit" name="lineEdit_bez"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Drehzahl</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_einvo"/>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="lineEdit_afb"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>[N]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Eintauchvorschub</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Ausräumen</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>[AFB]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>[VZ]</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Bezeichnung</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>[BEZ]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_ausr"/>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinBox_plm">
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Nutvorschub</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_drehz"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>[VXY]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>[PLM]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Punktlasermodi</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>[AR]</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>Ausführbedingung</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_vo"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_bild_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>461</width>
       <height>381</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_esc">
      <property name="text">
       <string>Abbruch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBox_wkz</tabstop>
  <tabstop>lineEdit_x</tabstop>
  <tabstop>lineEdit_y</tabstop>
  <tabstop>lineEdit_dm</tabstop>
  <tabstop>lineEdit_ti</tabstop>
  <tabstop>lineEdit_zust</tabstop>
  <tabstop>lineEdit_gegenlauf</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_esc</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>lineEdit_ausr</tabstop>
  <tabstop>lineEdit_einvo</tabstop>
  <tabstop>lineEdit_vo</tabstop>
  <tabstop>lineEdit_drehz</tabstop>
  <tabstop>spinBox_plm</tabstop>
  <tabstop>lineEdit_bez</tabstop>
  <tabstop>lineEdit_afb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
